Composer/packages/server/package.json

```diff
@@ -65,6 +65,7 @@
     "@microsoft/bf-dispatcher": "^4.10.0-preview.141651",
     "@microsoft/bf-generate-library": "^4.10.0-daily.20201008.172736",
     "@microsoft/bf-lu": "^4.11.0-dev.20201005.7e5e1b8",
+    "@microsoft/bf-orchestrator": "^4.11.0-beta.20201013.20d7917",
```

Comment:
"@microsoft/bf-orchestrator": "beta"

This will get latest from beta when dependencies are refreshed unless you want composer to pin the orchestrator library to a specific version?

Reply:
yes, in Composer even we specify the version here with "latest" it will always be pinned into a specific version, because we are using yarn.lock to save the version, which is considered as the best practise to make sure the product is stable.